charts/karmada/values.yaml

```diff
@@ -261,6 +261,9 @@ controllerManager:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param controllerManager.config.controllers the controllers launched by the manager.
+  config:
```

Comment:
I prefer to use `controllers` directly. `config` looks a bit redundant.

Reply:
I can use `controllers` directly instead of `config.controllers`, but I preferred the latter because it will be more organized if some additional parameters need to be added such as `cluster-status-update-frequency` or `v`...  
All these values will be clearly under `controllerManager.config` rather than directly at the root of `controllerManager`, which I find more readable.

What do you think about it ?

Reply:
I got what you mean. I prefer to put some related configuration items under the same group. It's ok to use `controllers` directly when there are currently no other options.
BTW, I don't think `config` is a good group name. All of them in `controllerManager` belong to the category of config.
```
## controller manager config
controllerManager:
```

Reply:
> I can use controllers directly instead of config.controllers, but I preferred the latter because it will be more organized if some additional parameters need to be added such as cluster-status-update-frequency or v...

I agree that we may need to customize other parameters later, they are organized together would be great!

> I don't think config is a good group name. All of them in controllerManager belong to the category of config.

How about `args`?

Reply:
Yes `args` seems like a better name to me.  
Should we use it as
- an object with predefined properties (like I did in the PR)
- an array where as many elements can be added freely. We could fill this variable like this during a release
```
controllerManager:
  args:
    - --controllers=-namespace,*
    - --v=3
```

The latter makes it possible to better cover all the possibilities without modification.

Reply:
I think the former is better. The latter may confuse users about which configuration items can be configured.

Reply:
The latter option allows to configure all args of `karmada-controller-manager`, that allows the user to configure anything he wants, not just what we imagine : there can always be unimagined scenarios, like for example mine. And  both current and future. And the chart will be compatible without change not only with current arguments but also with future ones.
 @RainbowMango What do you think about this two solutions ?

Reply:
> The latter option allows to configure all args of karmada-controller-manager, that allows the user to configure anything he wants, not just what we imagine

The idea is great, but my concern is the user experience. The users might need to specify all args at [here](https://github.com/karmada-io/karmada/blob/00b574f91c6ae3b030f90788e4f30f417bdabd21/charts/karmada/templates/karmada-controller-manager.yaml#L51-L56), even they just want to customize the `--controllers`.

Reply:
Kindly ping @fredgate .